index.js

```diff
@@ -1,3 +1,49 @@
 module.exports = {
-  extends: "eslint:recommended"
+  "parser": "babel-eslint",
```

Comment:
Also, I'm confused here. If you use this shared config, you'll get the babel-eslint parser, so is it optional as specified above?

Reply:
Good catch, I didn't mean to include that in the config.